src/native-common/Styles.ts

```diff
@@ -20,6 +20,13 @@ const forbiddenProps: string[] = [
     'cursor'
 ];
 
+// React Native styles that ReactXP doesn't expose.
```

Comment:
ReactXP doesn't expose `flexGrow`, `flexShrink`, or `flexBasis` to consumers of the library which means this change doesn't introduce any flex ambiguity issues.

However, this change does map `flex` to these three properties exactly the same way as ReactXP does on web. You are right that this will cause style adaptation overhead although I'm not familiar with the cost.

The reason that this mapping was introduced is that React Native changed the meaning of `flex`. The new meaning broke a bunch of layouts so I decided to map `flex` to grow, shrink, and basis the same way ReactXP does on web in order to maintain layout consistency between React Native and web. I can provide more details if you are interested.